custom_components/tuya_local/devices/arlec_pa1123bkha_portable_air_conditioner.yaml

```diff
@@ -0,0 +1,134 @@
+name: Arlec PA1123BKHA portable air conditioner

```

Comment:
Leave the branding off this top level name, in case other products match (due to the same device being sold in other markets with different branding). So just "Portable air conditioner" here, and "Arlec PA1123BKHA" in the products section below.